% !TeX root=../main.tex
% در این فایل، عنوان پایان‌نامه، مشخصات خود و چکیده پایان‌نامه را به انگلیسی، وارد کنید.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\latinuniversity{University of Tehran}
\latincollege{College of Engineering}
\latinfaculty{Faculty of New Science and Technology}
\latindepartment{Network}
\latinsubject{Information Technology}
\latinfield{Network Science}
%\latintitle{Inference of Tumor Phylogenetic Tree by Single Cell Mutations and CNV}
\latintitle{Reconstruction of Tumor Phylogenetic Tree based on SNV and CNV}
\firstlatinsupervisor{Dr. Saman Haratizadeh}
\secondlatinsupervisor{Dr. Abolfazl Motahari}
%\firstlatinadvisor{First Advisor}
%\secondlatinadvisor{Second Advisor}
\latinname{Afshin}
\latinsurname{Bozorgpour}
\latinthesisdate{Aug 2021}
\latinkeywords{Single-Nucleotide Variant, Copy Number Variation, Phylogenetic Tree, Reinforcement Learning, Deep Learning}
\en-abstract{
%Single-cell sequencing data have high potential in the reconstruction of evolutionary history of tumors, as rapid developments in single-cell sequence technology have developed in the past decade by designing different computational methods for inferring tumor trees with high precision.
%The primitive methods were trying to reach a tree with the highest score in terms of the specified data by searching for proper space in trees structures. In case the methods that were later presented were trying to reduce the search space by making their solution more powerful, using the mutation data matrix itself instead of trees.
%The observations that have recently been made reveal the relevance of changes to the number of copies with the evolutionary process of tumor cells, which makes it possible to form new assumptions that would not consider the assumption of infinite space model for tree construction as an comprehensive assumption.
%Hence, the data of changes in number of duplication at the side of SNV usage caused the formation of stronger assumptions that eventually led us to more accurate phylogenetic trees. In this thesis, we use both data for the reconstruction of the phylogenetic tree.
%In the proposed method, we use the data of replication variation at each stage of searching the tree with respect to the proposed response of genes that have the potential to be removed and considering this point, we evaluate the proposed response tree. We also replaced the simple search process that was previously completely random and inaccurate, through deep reinforcement learning networks with a powerful module for decision making that can be able to achieve a tree with minimal error following training with more accurate choices while reducing the search space. Also, after training the proposed network, the outputs of the primary method were compared with the basic method that had been completely random in the trees without using the data of the number of copies and using the assumption of infinite number of sites, indicating that the obtained responses were significantly higher in terms of convergence speed and the final accuracy.
}
