% !TeX root=../main.tex
% در این فایل، عنوان پایان‌نامه، مشخصات خود و چکیده پایان‌نامه را به انگلیسی، وارد کنید.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\latinuniversity{University of Tehran}
\latincollege{}
\latinfaculty{Faculty of New Sciences and Technologies}
\latindepartment{Network Group}
\latinsubject{Information Technology}
\latinfield{Networked System Engineering}
%\latintitle{Inference of Tumor Phylogenetic Tree by Single Cell Mutations and CNV}
\latintitle{Reconstruction of Tumor Phylogenetic Tree based on SNV and CNV}
\firstlatinsupervisor{Dr. Saman Haratizadeh}
\secondlatinsupervisor{Dr. Abolfazl Motahari}
%\firstlatinadvisor{First Advisor}
%\secondlatinadvisor{Second Advisor}
\latinname{Afshin}
\latinsurname{Bozorgpour}
\latinthesisdate{Aug 2021}
\latinkeywords{Phylogenetic Tree, Single-Cell Sequencing, Single-Nucleotide Variant, Copy Number Variation, Reinforcement Learning, Deep Learning}
\en-abstract{
Single-cell sequencing data contains excellent information for the treatment of cancer. These data have a very high ability to reconstruct the evolutionary history of tumors.
In recent years, the advances that have been made in this field, it has paved the way for the inference of the phylogenetic tree with high accuracy, which is one of the most important pillars in the treatment of cancer.
The primitive methods tried to build a tree with classical approaches such as reducing distance and adjacent connection. Subsequent methods tried to reach the tree with the highest score according to their observations by searching conveniently in the state space of the trees while the methods presented later tried to reduce the search space by making their method more powerful and do this search in the space of the mutation data matrix instead of the tree.
Recent observations show that changes in the number of copies are associated with the evolution of tumor cells, leading to new hypotheses that do not make the model of infinite locations a complete assumption for tree construction. Thus, the use of copy number variation data along with single nucleotide variants gave rise to stronger hypotheses that ultimately led us to phylogenetic trees with greater accuracy. In this thesis, both data have been used to construct a phylogenetic tree.
In the proposed method, using copy number abbreviation at each stage of the tree search, we identify genes for which there is a potential for loss, and according to this point, we evaluate the proposed response tree.
We also replaced the simple search process, which was previously completely random and unintelligent, with deep learning enhancement networks with a powerful decision-making module.
This module can reduce the search space after proper training with more accurate choices. It also provides the ability to reach a tree with a minimal error by reducing the number of steps to achieve a great response.
The outputs of the proposed method are compared with the results of the basic method, which, assuming the model of infinite site assumption and performed the search in a non-intelligent and completely random manner in the space of trees. The results show that the performance of the proposed method in terms of both convergence speed and final accuracy has a significant improvement against the previous case, which can infer the phylogenetic tree with higher accuracy for its observations. 
}
