% !TeX root=../main.tex
% در این فایل، عنوان پایان‌نامه، مشخصات خود و چکیده پایان‌نامه را به انگلیسی، وارد کنید.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\latinuniversity{University of Tehran}
\latincollege{College of Engineering}
\latinfaculty{Faculty of New Science and Technology}
\latindepartment{Network}
\latinsubject{Information Technology}
\latinfield{Network Science}
\latintitle{Inference of Phylogenetic Tree for Inter Tumor using Single Cell Mutations and CNV}
\firstlatinsupervisor{Dr. Saman Haratizadeh}
\secondlatinsupervisor{Dr. Abolfazl Motahari}
%\firstlatinadvisor{First Advisor}
%\secondlatinadvisor{Second Advisor}
\latinname{Afshin}
\latinsurname{Bozorgpour}
\latinthesisdate{Aug 2021}
\latinkeywords{Single-Nucleotide Variant, Copy Number Variation, Phylogenetic Tree, Reinforcement Learning, Deep Learning}
\en-abstract{
%Single-cell sequencing data have great potential in reconstructing the evolutionary history of tumors, so that rapid advances in single-cell sequencing technology over the past decade have provided the basis for inferring tumor progeny trees with high accuracy by designing various computational methods. The primitive methods tried to reach the tree with the highest score according to the specified data by searching properly in the state space of the trees. If the methods presented later tried to reduce the search space by using their method more powerfully and use the mutation data matrix itself instead of the tree. Recent observations show that changes in the number of copies are associated with the evolution of tumor cells, leading to new hypotheses that do not make the model of infinite locations a complete assumption for tree construction. Thus, the use of copy number variation data along with monoclonal mutations gave rise to stronger hypotheses that ultimately led us to phylogenetic trees with greater accuracy. In this dissertation, both data have been used to construct a phylogenetic tree. In the proposed method, using duplicate copy change data at each stage of the tree search according to the proposed response, we identify genes for which there is a potential for deletion, and according to this point, we evaluate the proposed response tree. We also replaced the simple search process, which was previously completely random and unintelligent, with the use of deep learning reinforcement networks with a powerful decision-making module that can, after training, make more accurate choices while reducing search space, leading to Reduce the steps to achieve the desired answer, provide the ability to reach a tree with minimal error. Also, after training the proposed network, the obtained outputs are compared with the basic method, which completely randomly searches the space of possible trees without the use of copy number change data and using the assumption of infinite locations, which shows the obtained answers. The results have a significant advantage both in terms of convergence speed and final accuracy. 
}
